reference/OpenAPI.json

```diff
@@ -35008,11 +35013,13 @@
         "type": "object",
         "title": "Campaigns Delete Response Body",
         "description": "Response body schema for **DELETE** `/campaigns/{campaignId}`.",
-        "allOf": [
-          {
-            "$ref": "#/components/schemas/AsyncActions"
+        "properties": {
```

Comment:
Dlaczego properties skoro jest AsyncActions? Jest tutaj jakiś problem, że nie zawsze jest required?

Reply:
tak jest

Reply:
TODO, revert - change in campaigns delete definition response